packages/roosterjs-plugin-picker/lib/PickerPlugin.ts

```diff
@@ -112,9 +111,7 @@ export default class PickerPlugin<T extends PickerDataProvider = PickerDataProvi
     public willHandleEventExclusively(event: PluginEvent) {
         return (
             this.isSuggesting &&
-            (event.eventType == PluginEventType.KeyDown ||
-                event.eventType == PluginEventType.KeyUp ||
-                event.eventType == PluginEventType.Input)
+            (event.eventType == PluginEventType.KeyDown || event.eventType == PluginEventType.KeyUp)

```

Comment:
I removed `PluginEventType.Input` because this is the event CustomReplace needs to listen to. Looking at the old EmojiPlugin and AtMentionsPlugin, `KeyDown` and `KeyUp` seemed sufficient. @Lego6245 let me know if Input needs to be added back in.

Reply:
A little concern about this.
But let me think about it.

Reply:
Let's just focus on handle keypress event in this PR.